public/locales/en/translations.json

```diff
@@ -2,11 +2,7 @@
   "companyName": "Contoso",
   "header": {
     "appName": "Azure IoT Remote Monitoring",
-    "logout": "Logout",
-    "profile": "Profile",
-    "getStarted": "Get started",
```

Comment:
Please go through and double check the merge with master, these strings should now be removed.